docker-compose.yml

```diff
@@ -149,9 +166,6 @@ services:
       - COURSIER_CACHE=/opt/raster-foundry/app-backend/.coursier-cache
       - RF_LOG_LEVEL=INFO
       - SBT_OPTS="-Xmx2G -XX:MaxPermSize=2G"
-    ports:
-      - "8080:8080"
-      - "9030:9020"
```

Comment:
Removing this port makes the JMX arguments in `command` impossible to interact with. I think that we likely want to keep this port mapping definition.

Reply:
I created the ECR repositories and pushed the containers up. I was going to do that out-of-band once the PR was approved so the images we push up would be solid. 

I added the port mapping back in.